README.md

```
@@ -1,6 +1,11 @@
 xmltree-rs
 ==========
 
+> [!WARNING]
+> This is a fork of the "real" `xmltree` with support for attribute namespaces. For more information, see this issue: https://github.com/eminence/xmltree-rs/issues/13
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)